src/ocean/text/convert/Formatter_test.d

```diff
@@ -424,9 +424,19 @@ unittest
     }
 
     Foo f = Foo.FooBar;
-    test("42" == format("{}", f));
+    test("Foo.FooBar" == format("{}", f));
     f = cast(Foo)36;
-    test("36" == format("{}", f));
+    test("cast(Foo) 36" == format("{}", f));
+    f = cast(Foo)0;
+    test("Foo.A" == format("{}", f));
+
+    enum FooA : string { a = "alpha", b = "beta" }
+    FooA fa = FooA.a;
+    test("FooA.a" == format("{}", fa));
+    fa = cast(FooA)"beta";
+    test("FooA.b" == format("{}", fa));
+    fa = cast(FooA)"gamma";
+    test("cast(FooA) gamma" == format("{}", fa));
```

Comment:
Please also add test for non-allocation (see `testNoAlloc` examples)

Reply:
That's a good suggestion.